content/en/docs/Installation/_index.md

```diff
@@ -218,6 +218,8 @@ ClusterRoles used by Kyverno:
 
 The `generate` rule creates a new resource, and to allow Kyverno to create resources the Kyverno ClusterRole needs permissions to create/update/delete. This can be done by adding the resource to the ClusterRole `kyverno:generatecontroller` used by Kyverno or by creating a new ClusterRole and a ClusterRoleBinding to Kyverno's default ServiceAccount.
 
+To get cluster wide permissions, users has to add the permissions for cluster wide resource such as (`roles, clusterroles, rolebindings, clusterrolebindings`) they need.
```

Comment:
"users must add..."

Reply:
`roles`, `clusterroles`, `rolebindings`, and `clusterrolebindings` should each be enclosed in back ticks, not the whole series enclosed in a single pair.